training/src/anemoi/training/commands/mlflow.py

```diff
@@ -104,7 +148,7 @@ def run(args: argparse.Namespace) -> None:
                 raise ValueError(msg)
 
             TokenAuth(url=url).login(force_credentials=args.force_credentials)
-            return
+            return None
```

Comment:
Why all the explicit None returns? Is the linter requiring this? The return here is just to break the function early, not the return a value to a caller. So could you change these to be just empty returns.

Reply:
Yes I agree but pre-commit is forcing my hand. Is there anyway I can bypass it?

Reply:
I'm trapped in a fight I can not win:
```
ruff.....................................................................Failed
- hook id: ruff
- exit code: 1
- files were modified by this hook

warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `training/pyproject.toml`:
  - 'ignore' -> 'lint.ignore'
  - 'select' -> 'lint.select'
Found 1 error (1 fixed, 0 remaining).
```
I think it's applying this one: https://docs.astral.sh/ruff/rules/implicit-return/ but this one https://docs.astral.sh/ruff/rules/unnecessary-return-none/ should be applied, right?

Reply:
@gmertes As you suggested, removing the unwanted return value fixed the linter issue!